dfastmi/batch/FileNameRetriever.py

```diff
@@ -27,121 +27,210 @@
 This file is part of D-FAST Morphological Impact: https://github.com/Deltares/D-FAST_Morphological_Impact
 """
```

Comment:
Because this module contains multiple classes would the following comment / header info handy?
```suggestion
"""
"""
Module for Abstract class AFileNameRetriever, FileNameRetriever implementations and the FileNameRetrieverFactory classes

Abstract Class:
    AFileNameRetriever
    
Class: 
    FileNameRetrieverUnkown
    FileNameRetrieverLegacy
    FileNameRetriever

Static Class: 
    FileNameRetrieverFactory

"""
```

Reply:
agreed